pkg/cli/server/root_test.go

```diff
@@ -48,6 +48,12 @@ func TestServe(t *testing.T) {
 	})
 
 	Convey("Test serve config", t, func(c C) {
```

Comment:
Also add a test case for more than one arg

Reply:
Added the test case for 2 args